kurtosis-module/impl/module_io/params.go

```diff
@@ -89,6 +89,17 @@ type ExecuteParams struct {
 	// If set, the module will block until a finalized epoch has occurred
 	WaitForFinalization bool	`json:"waitForFinalization"`
 
+	// If set, the module will block until all verifications have passed
+	WaitForVerifications bool `json:"waitForVerifications"`
+
+	// If set, verifications will be marked as failed if the TTD is not reached within specified number of epochs
+	// Default: 5
+	VerificationsTTDEpochLimit *uint64 `json:"verificationsTTDEpochLimit"`
```

Comment:
Suggestion: if you make this `uint64` rather than `*uint64`, then it'll be easier to supply the default params in `GetDefaultExecuteParams()`